catalogs/catalog-jdbc-doris/src/test/java/com/datastrato/gravitino/catalog/doris/integration/test/TestDorisDatabaseOperations.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.doris.integration.test;
+
+import com.datastrato.gravitino.utils.RandomNameUtils;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("gravitino-docker-it")
+public class TestDorisDatabaseOperations extends TestDorisAbstractIT {
+
+  @Test
+  public void testBaseOperationDatabase() {
+    String databaseName = RandomNameUtils.genRandomName("ct_db");
+    Map<String, String> properties = new HashMap<>();
+    String databaseComment = "test_comment";
+
+    Assertions.assertThrows(
+        UnsupportedOperationException.class,
+        () -> {
+          DATABASE_OPERATIONS.create(databaseName, databaseComment, properties);
```

Comment:
It seems a UT not an e2e test, but you put this in the package `integration.test`

Reply:
It's normal, currently, we don't differentiate between IT and UT very much.  The package `integration.test` holds tests that rely on the docker environment.

Reply:
Yeah, `TestDorisDatabaseOperations` relies on docker environment but it's a unit test actually.

Reply:
Therefore, I do not recommend putting this class in the package `integration.test`

Reply:
In other projects (like Apache kyuubi), they also use Docker container in unit test. so I think it's ok here.

https://github.com/apache/kyuubi/blob/master/externals/kyuubi-jdbc-engine/src/test/scala/org/apache/kyuubi/engine/jdbc/doris/WithDorisContainer.scala

Reply:
> Therefore, I do not recommend putting this class in the package `integration.test`

At this point, I discussed it with @jerryshao in #2279 , so it is acceptable for the package "integration.test" to hold tests that are dependent upon the docker environment.

Reply:
We will use another issue https://github.com/datastrato/gravitino/issues/2599 to track it, @zhoukangcn please go ahead.